Utilities/CodeGeneration/EdFi.Ods.CodeGen/Properties/launchSettings.json

```diff
@@ -2,7 +2,7 @@
   "profiles": {
     "EdFi.CodeGen - SQLServer": {
       "commandName": "Project",
-      "commandLineArgs": "-e SQLServer"
+      "commandLineArgs": "-e SQLServer -r C:\\oss-workspace\\ -i -t"
```

Comment:
Don't hard code your workspace in here, and preferably don't default to use plugins.  See also discussion around the code gen changes and your new parameter.

Reply:
I removed it my changes